src/goto-instrument/cover.cpp

```diff
@@ -315,9 +341,11 @@ void instrument_cover_goals(
   goto_model_functiont &function,
   message_handlert &message_handler)
 {
+  const symbolt *function_symbol =
+    function.get_symbol_table().lookup(function.get_function_id());
```

Comment:
Use `lookup_ref` to avoid storing in a pointer that you immediately dereference.